src/ngraph/op/scatter_add.cpp

```diff
@@ -78,3 +78,26 @@ void op::ScatterAdd::validate_and_infer_types()
 
     set_output_type(0, inputs_et, inputs_shape);
 }
+
+//------------------------------------------------------------------------------
+//
+//          Opset version 3
```

Comment:
In fact it's not a different opset here but a different op version.

Reply:
How about "Introduced in Opset 3"?